Tensile/Source/lib/include/Tensile/Utils.hpp

```diff
@@ -152,6 +152,31 @@ namespace Tensile
         return stream;
     }
 
+    constexpr uint32_t bitsNeeded(uint32_t maxVal)
```

Comment:
This function name could use more context.  Maybe these functions could be grouped into a namespace or class?